mgmt/WebMgmtUtils.cc

```diff
@@ -226,16 +236,21 @@ varSetInt(const char *varName, RecInt value, bool convert)
   case RECD_INT:
     RecSetRecordInt((char *)varName, (RecInt)value, REC_SOURCE_EXPLICIT);
     break;
+
   case RECD_COUNTER:
     if (convert) {
       RecSetRecordCounter((char *)varName, (RecCounter)value, REC_SOURCE_EXPLICIT);
       break;
     }
+  // fallthrough
+
```

Comment:
Reading through the code we want to set the found = false if it can't convert the value.  This is what other methods do.  Also, these methods are not used.